src/deluge/hid/encoders.cpp

```diff
@@ -151,12 +151,6 @@ bool interpretEncoders(bool inCardRoutine) {
 					                                   Buttons::isButtonPressed(deluge::hid::button::TEMPO_ENC),
 					                                   Buttons::isShiftButtonPressed());
 				}
```

Comment:
removed as its no longer required. there was previously a bug where the popup when turning tempo encoder remained visible, as that got fixed, it is no longer required to remove it in the automation view.